src/porepy/models/run_convergence_study.py

```diff
@@ -0,0 +1,217 @@
+import os
+import logging
+from typing import (  # noqa
+    Any,
+    Callable,
+    Coroutine,
+    Generator,
+    Generic,
+    Iterable,
+    List,
+    Mapping,
+    Optional,
+    Set,
+    Tuple,
+    Type,
+    TypeVar,
+    Union,
+)
+from pathlib import Path
+from pprint import pformat
+import functools
+import inspect
+
+import pendulum
+import porepy as pp
```

Comment:
all these import are standard or we need to install new modules? it looks like you use a lot of modules never used in pp

Reply:
Good point. This module should not be imported in pp.__init__ - that is probably not relevant anyhow, and usage and installation of the new modules should be documented. is it pip

Reply:
I think some of the imports could be avoided.